specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/job/getJobRunbookContent.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "subscriptionId": "51766542-3ed7-4a72-a187-0c8ab644ddab",
+    "resourceGroupName": "mygroup",
+    "automationAccountName": "ContoseAutomationAccount",
+    "jobName": "foo",
+    "api-version": "2023-05-15-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Content-Type": "text/powershell"
+      },
+      "body": "get-date"
```

Comment:
>    "body": "get-date" [](http://example.com/codeflow?start=2&length=22)

Having a string response is not easy to extend in the future without breaking clients. I'd suggest you change it to a JSON response unless there's a good reason not to

(And I can't help feeling curious why this would be the result...)

Reply:
This is existing changes which are already shipped , we would consider this for any new changes.